server/ServerFSM.cpp

```diff
@@ -452,15 +452,19 @@ void ServerFSM::UpdateIngameLobby() {
     }
     dummy_lobby_data.m_start_lock_cause = UserStringNop("SERVER_ALREADY_PLAYING_GAME");
 
-    // send it to all those without empire
-    // and who are CLIENT_TYPE_HUMAN_PLAYER
+    auto player_info_map = m_server.GetPlayerInfoMap();
```

Comment:
Can / should this be a const reference `const auto&` ?

Reply:
No, it cann't. `GetPlayerInfoMap` doesn't return existing map but builds new one and return it by value.